src/components/node/sul-study-place/study-place-features.tsx

```diff
@@ -42,21 +48,30 @@ const StudyPlaceFeatures = ({branchHours, branchTitle, branchUrl, capacity, cont
 
       <div className="card-body items-start rs-px-3 rs-pb-3 rs-pt-7 md:rs-pt-3 w-full">
         <div className="leading-display text-18 pt-0 font-normal ">
-          <h2 id={headingId} className="type-3 rs-mb-1">{type}</h2>
+          <h2 id={headingId} className="type-3 rs-mb-1">{roomDonorName} {type}</h2>
```

Comment:
need to trim if the donor name is empty.

Reply:
@pookmish How about `[roomDonorName, type].join(" ")` instead? That way it ignores the space if null.

Reply:
You'd have to filter it if you do that. `[roomDonorName, type].filter(item => !!item).join(" ")`

Reply:
Hm. Not a fan of that. I could trim it this way: `{(node.sulStudyRoomDonorName || '').trim()} {node.sulStudyType.name}`

Reply:
You still have the space between. You're trimming the wrong thing. Something like this would work:
```
{node.sulStudyRoomDonorName ? `${node.sulStudyRoomDonorName} ${node.sulStudyType.name}` : node.sulStudyType.name}}
```

Reply:
@pookmish Okay. Sorry. I went back to the join version. Just seems a bit easier for me to know what's going on. 

It's interesting that the space never seemed to appear in the UI. I could only see it in console.log(). Which I admit I probably should have ran it through sooner.

Reply:
Honestly, it might not even make any difference. But I feel it's good practice.

Reply:
I also experienced a weird build issues. Both on my local and with the build on this PR. I finally had to switch to another ticket out of frustration. When I came back, it stopped being an issue. Been trying to replicate it. Hope fully it's just one of those things.

Reply:
interesting, Using the `.filter` produces this dom markup: `<h2 class="type-3 rs-mb-1">Group study room</h2>`
While just printing the two values next to eachother produces this dom: `<h2 class="type-3 rs-mb-1"> <!-- -->Group study room</h2>`

Reply:
The filter way seems cleaner.